packages/amplication-client/src/VersionControl/Commit.tsx

```diff
@@ -62,7 +61,7 @@ const Commit = ({ projectId, resourceId, noChanges }: Props) => {
       {
         query: GET_BUILDS_COMMIT,
         variables: {
-          resourceId,
+          resourceId: "",
```

Comment:
why ?

Reply:
hum, we forgot about it. Can we remove it?